.npmignore

```diff
@@ -0,0 +1,3 @@
+.gitignore
```

Comment:
[Sequelize](https://github.com/sequelize/sequelize/blob/master/.npmignore) for example does not include that line and the package does not contain the `.gitignore` file.

You can test it by: 
 - removing the line
 - `npm pack`
 - `tar -xzf eslint-config-seegno-7.0.0.tgz`
 - `ls -la package`